docs/source/specs/openapi.json

```diff
@@ -510,6 +548,25 @@
         ],
         "summary": "List the roles for a tenant",
         "operationId": "listRoles",
+        "parameters": [{
+            "name": "page",
```

Comment:
@chambridge we can also add a reusable parameter here similar to 
https://github.com/ManageIQ/service_portal-api/blob/d6b58bfdaedf928e3574d61211fe9cd7f0973284/public/service_portal/v0.1.0/openapi.json#L820
and then reuse it with a single ref like here
https://github.com/ManageIQ/service_portal-api/blob/d6b58bfdaedf928e3574d61211fe9cd7f0973284/public/service_portal/v0.1.0/openapi.json#L735
So its defined just once